### ISSUE TITLE
Troubles with first method in class 

### ISSUE THREAD

**Issue by kud** [babel/babel-sublime#330]
![image](https://cloud.githubusercontent.com/assets/655838/26719441/d7ecfc8e-4784-11e7-8286-d5120c43fe94.png)
![image](https://cloud.githubusercontent.com/assets/655838/26719449/e0770a0c-4784-11e7-8f2a-6823bbdb0bd0.png)

It should be blue and not white.

Related: https://github.com/SintrumIT/oceanic-next-italic/issues/25

test:

```
class Division extends React.Component {
  render() {
  }
}

export default Division
```

```
class Division extends React.Component {
  state = {}

  render() {
  }
}

export default Division
```

**Comment by ssangervasi** [comment]
I have encountered this problem with methods that are defined using the property-initializer syntax. I know this syntax is experimental, but it is very useful for automatically binding callbacks to a react component.

Here is an example that shows all the scopes that are applied to three different methods in a class:
```js
class ClassWithBoundMethods {
  /**
   * Sublime scopes (fine):
   *  source.js
   *  meta.class.js
   *  meta.class-method.js
   *  string.unquoted.js
   *  entity.name.function.js
   */
  classMethod() {}

  /**
   * BROKEN Sublime scopes:
   *  source.js
   *  meta.class.js
   *  meta.class.property.js
   *  variable.other.property.js
   */
  firstBoundMethod = () => { return this; }

  /**
   * Sublime scopes (fine):
   *  source.js
   *  meta.class.js
   *  meta.class.property.js
   *  meta.function.arrow.js
   *  entity.name.function.js
   */
  secondBoundMethod = () => { return this; }
}
```

Note that `firstBoundMethod` and `secondBoundMethod` are identical except for the order in which they appear. They should be scoped the same, but are not.

Also interesting: if you add a semicolon anywhere between the two bound methods, then `secondBoundMethod` becomes broken as well.

**Comment by borela** [comment]
Try FJSX15 https://github.com/borela/naomi it supports classes, properties and even stage-0 features.

**Comment by Thom1729** [comment]
This should be fixed in v10.0.0-beta.1. See #379 for more information on the new release. Please report any problems with v10 in a new issue.
